### PR TITLE
Add http prefix

### PR DESCRIPTION
**Pull request by YazzyYaz**
To merge when https://github.com/celestiaorg/celestia-node/pull/768 is merged

### CODE REVIEW

File: docs/developers/node-tutorial.md [hunk 1]
```diff
@@ -53,14 +53,14 @@ an example public Core Endpoint.
...
-Here we are starting a light node with a connection to a Core endpoint at
-`https://api-mamaki.pops.one/` and also telling the light node to use the `developer`
-key we generated as its default account.
+Here we are starting a light node with a connection to a Core endpoint and
+also telling the light node to use the `developer` key we generated
+as its default account.
 
 ```sh
-celestia light start --core.grpc https://api-mamaki.pops.one/ --keyring.accname developer
+celestia light start --core.grpc http://<ip-address>:9090 --keyring.accname developer
```
Comment (liamsi) [inline docs/developers/node-tutorial.md:63]: Is that port always the same or is configurable?

Comment (YazzyYaz) [reply docs/developers/node-tutorial.md:63]: good question, I've seen examples of someone use port 8090 but I think the default is 9090 cc @renaynay @Wondertan @Bidon15

Comment (renaynay) [reply docs/developers/node-tutorial.md:63]: 9090 is the default port grpc for celestia-app @liamsi 

We can substitute 9090 for <port> here (which I recommend) but also mention in comments that 9090 is the default for celestia-app so it's likely it's open on 9090

Comment (Bidon15) [reply docs/developers/node-tutorial.md:63]: It is configurable in app.toml
if the app.toml in celestia-app node instance is left as default, then 9090.

**Comment by adlerjohn** [comment]
Converting to draft until unblocked

**Comment by YazzyYaz** [comment]
@adlerjohn given the other PR is merged, can this be reviewed?

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**